libavformat/imfdec.c

```diff
@@ -585,6 +589,13 @@ static int set_context_streams_from_tracks(AVFormatContext *s)
             av_log(s, AV_LOG_ERROR, "Could not copy stream parameters\n");
             return ret;
         }
+
+        ret = av_dict_copy(&asset_stream->metadata, first_resource_stream->metadata, 0);
```

Comment:
Required to copy the global DoVi metadata that belongs to the stream as a whole.